cves/kernel/CVE-2017-18549.yml

```diff
@@ -475,5 +524,5 @@ nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it.
```

Comment:
Can't comment on unchanged code, so I'm putting this here:

Line 520: Make sure you confirm the CWE listed.

Reply:
CWE is already listed.